### PR TITLE
Close the file handler if the file is corrupt.

### PR DESCRIPTION
**Pull request by fujiisoup**
Trying to address #41

### CODE REVIEW

File: testings/test_open.py [hunk 1]
```diff
@@ -49,6 +51,32 @@
...
+        check_output(["grep",filename], stdin=lsout.stdout, shell=False)
+
+        for proc in psutil.process_iter():
+            try:
+                for item in proc.open_files():
+                    if filename == item.path:
+                        return True
+            except Exception:
+                pass
+        raise Exception("File is in use.")
```
Comment (fujiisoup) [inline testings/test_open.py:69]: @willberriss

Here is the line where I tried to make the test fail, if the file is in use.

Comment (fujiisoup) [reply testings/test_open.py:69]: I found it! I did opposite.

**Comment by fujiisoup** [comment]
Merged.

Thank you,@willberriss, for your help!!

**Comment by willberriss** [comment]
Thank you for fixing it @fujiisoup!

**Pull request merged**